content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -180,9 +197,28 @@ All code snippets in Windows sections are to be run in a PowerShell environment
     [Environment]::SetEnvironmentVariable("HTTPS_PROXY", "http://proxy.example.com:443/", [EnvironmentVariableTarget]::Machine)
     ```
 
-    If after reboot you see the following error, you need to restart the docker service manually
+    After reboot, you can check via below command to check your docker service is ready.
 
-    ![alt_text](../windows-docker-error.png "windows docker error screen capture")    
+    ```PowerShell
+    docker version
+    ```
+
+    If you see error message like the following, you need to restart the docker service manually.
+
+    ```dummy
```

Comment:
Replace `dummy` with the empty string here.

Reply:
Okay, I'll apply.